docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -1161,6 +1161,28 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
             Scripts</link>) for more details.</para>
           </listitem>
         </orderedlist>
+
+        <sect3 id="Addl_SSH_KeyInfo">
+          <title>Additional SSH Key Information</title>
+
+          <para>On a multi-node HPCC system certificates and SSH keys must all
+          match across all nodes in order for the system to work properly. If
+          you used the <emphasis>install-cluster.sh</emphasis> script as
+          outlined in the steps above, this would insure that everything is
+          properly in sync. However, it is still a good idea to verify that
+          they do all match up. Another way to ensure this is the case is to
```

Comment:
Another way to ensure this is the case is to
    sb
Another way to ensure this is to